tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -156,6 +156,7 @@ struct MklConvFwdParams {
     string name;
     mkldnn::algorithm alg;
     std::vector<float> param;
+    std::string param_key;
```

Comment:
I think the name `param_key` is a bit redundant since it's already a property of a param parameter. Maybe rename it to `partial_key` to be more descriptive? (Up to you.)

Reply:
Fixed